test/test_web.rb

```diff
@@ -663,4 +663,46 @@ def app
       assert_equal 'nicehost.org', session_options[:host]
     end
   end
+
+  describe 'sidekiq web sessions options' do
+    include Rack::Test::Methods
+
+    def app
+      Sidekiq::Web.new
+    end
+
+    after { Sidekiq::Web.enable(:sessions) }
```

Comment:
@inkstak I agree with you, but they are not for the sole purpose of testing.

The ideal design is to have only instance methods actually, the class methods only exist to provide retrocompatibiity with the old implementation, maybe we can change that in the future, depending on how @mperham sees it.

The issue with configuring `Sidekiq::Web` directly is that this change will replicate to the subsequent tests, unless you undefine the constant and reload the code.